### ISSUE TITLE
Doubts regarding the PLSTM.py code

### ISSUE THREAD

**Issue by Anirudh257** [FesianXu/PLSTM#5]
Hey, sorry for bothering you again. I had some more doubts regarding the PLSTM code:

1) You have assigned *g_feat_dim = 25 * 3* but aren't their cases in the NTU dataset when more than 1 skeleton is present. Shouldn't we have 50 * 3 as the value?

2) In **g_num_action = 40**, doesn't the NTU dataset have 60 action labels? Why have we considered this as 40 only then?

3) On line 53, 
>      loss_summary_list = [action_reg_loss_s, action_raw_loss_s, action_pred_acc_s]
why is the **action_pred_acc_s** considered to be a loss? 
What is the intuition behind 2 losses? **action_reg_loss** and **action_raw_loss**? One is calculating the cross-entropy and the other is adding the losses found element-wise?

**Comment by FesianXu** [comment]
Hi anirudh

A1 & A2:
Actually i have evaluated my code on two database, one is NTU and another is UESTC[1]. And the parameters like the `num_actions` or `g_feat_dim` are still remain on the config of UESTC. So, yes of course, you can modify those parameters (Sorry for not noting those information at first and cause confusion.)

A3: In fact, it's a list prepared for the TensorBoard to visualize the loss and metric. The metric like accuracy won't have any gradient and thus doesn't need to be trained. And I just make a mistake in name the list, which confused you. I'm sorry for the mistake.

I hope my answers can solve your problems. :)

Reference:
[1].   https://github.com/HRI-UESTC/CFM-HRI-RGB-D-action-database

**Comment by Anirudh257** [comment]
Thanks, again for clarifying my doubts :)
